setup.py

```diff
@@ -12,8 +12,14 @@
           'Intended Audience :: Developers',
           'License :: OSI Approved :: Zope Public License',
           'Operating System :: OS Independent',
+          'Programming Language :: Python :: 2',
           'Programming Language :: Python :: 2.7',
-          'Programming Language :: Python :: 2 :: Only',
+          'Programming Language :: Python :: 3',
+          'Programming Language :: Python :: 3.5',
```

Comment:
Python 3.5 is in EOL state since Sept 2020.

Reply:
Yeah, I forgot to remove this line :+1: Done now!